bin/MadGraph5_aMCatNLO/cards/production/2017/QCD_HT_LO_MLM/QCD_HT2000toInf/QCD_HT2000toInf_proc_card.dat

```diff
@@ -0,0 +1,17 @@
+import model sm-ckm_no_b_mass
+define p = g u c d s u~ c~ d~ s~
```

Comment:
remove p and j definitions, remove duplicated lepton definitions

Reply:
(p and j definitions are automatically taken into account when importing a given model)